src/System.Private.Xml.Linq/src/System/Xml/Linq/XNodeBuilder.cs

```diff
@@ -68,7 +68,7 @@ public override void WriteCData(string text)
 
         public override void WriteCharEntity(char ch)
         {
-            AddString(new string(ch, 1));
+            AddString(char.ToString(ch));
```

Comment:
I saw that you did `ch.ToString` in the other PR, is it equivalent?

Reply:
They're equivalent: `ch.ToString()` just calls `char.ToString(ch)`.  But I'll change the other one.